rakefile-dotnet.rb

```diff
@@ -258,6 +268,12 @@
   cmd.parameters "pack #{nuspec} -Version #{version}#{version_suffix} -OutputDirectory #{output}"
 end
 
+desc "create the nuget package with the .NET Standard"
+exec :pack => [:build, output] do |cmd|
+  cmd.command = nuget_command
+  cmd.parameters "pack #{dotnet_nuspec} -Version #{version}-dotnet#{version_suffix} -OutputDirectory #{output}"
+end
+

```

Comment:
The problem with that versioning scheme is that `netstd` sorts after beta, so if we're on 2.3.0 net40 and 2.4.0-netstd net40+netstd then we can't push out 2.4.0-beta net40 when we get closer to release because because it's already trumped by 2.4.0-netstd.